### PR TITLE
README Refactoring

### PR DESCRIPTION
**Pull request by 0xMRTT**

### CODE REVIEW

**Review by tfuxu — COMMENTED** [review]
Looks nice, but we should bring back the `Theming Setup` section, or turn it into another section, maybe a FAQ section?

**Comment by daudix-UFO** [comment]
I don't know what to do, keep pretty much same instructions in docs and readme or add link to docs only

**Review by daudix-UFO — APPROVED** [review]

**Comment by 0xMRTT** [comment]
@daudix-UFO could you check my review 🤗

**Comment by daudix-UFO** [comment]
> @daudix-UFO could you check my review hugs

Wdym?

**Review by daudix-UFO — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -127,9 +128,6 @@ ninja -C builddir run
 
 ## 📦️ Alternative installation methods
 
-> **Warning**
```
Comment (daudix-UFO) [inline README.md]: Why you removed this?

File: README.md [hunk 2]
```diff
@@ -85,13 +85,14 @@ environment cross-distro. Thus, it is highly recommended you use it.
...
 The following packages are required to build Gradience:
 
-* python3
-* python-gobject
-* [blueprint-compiler](https://jwestman.pages.gitlab.gnome.org/blueprint-compiler/setup.html)
-* gtk4
-* libadwaita (>= 1.2)
-* meson
-* ninja-build
+- Python 3 `python`
```
Comment (daudix-UFO) [inline README.md:88]: Why `-` and not `*`?

File: README.md (hunk 1 above)
Comment (tfuxu) [reply README.md]: ah, sorry I didn't meant to remove that

File: README.md (hunk 2 above)
Comment (tfuxu) [reply README.md:88]: they are the same in markdown

Comment (daudix-UFO) [reply README.md:88]: Also why putting name and name of package instead of just package name?

Comment (daudix-UFO) [reply README.md:88]: > they are the same in markdown

So why you changed them?

Comment (tfuxu) [reply README.md:88]: because, not every distro have the same package name, and it just looks better

Comment (daudix-UFO) [reply README.md:88]: Alright, keeping it

Comment (daudix-UFO) [reply README.md:88]: > it just looks better

This is the point of refactoring :D

Comment (tfuxu) [reply README.md:88]: for example pygobject in Ubuntu is packaged with a name `python3-gi` and in Arch Linux it is `python-gobject`

**Review by tfuxu — APPROVED** [review]

**Comment by daudix-UFO** [comment]
New README in 3… 2…

**Comment by daudix-UFO** [comment]
1…

**Comment by 0xMRTT** [comment]
The review of 0xMRTT has been skipped 😳

**Comment by daudix-UFO** [comment]
You not added yourself to reviewers :///

**Comment by 0xMRTT** [comment]
I mean that I said that the proof of concept could be changed because it's a bit old

**Comment by daudix-UFO** [comment]
Also not sure about vanilla AUR setup as most users just use AUR helpers like `yay` or `paru`

**Comment by daudix-UFO** [comment]
> I mean that I said that the proof of concept could be changed because it's a bit old

Can you explain?

**Comment by 0xMRTT** [comment]
The picture is a screenshot from artif from the first version of Gradience I thought about using a more actual screenshot

**Comment by daudix-UFO** [comment]
I can't take one as my screen is very small, can you do this?

**Comment by 0xMRTT** [comment]
I can try

**Pull request merged**